bedrock/mozorg/models.py

```diff
@@ -129,3 +131,42 @@ def blog_title(self):
     @property
     def blog_link(self):
         return settings.BLOG_FEEDS[self.blog_slug]['url']
+
+    @classmethod
+    def parse_feed(cls, feed_id, feed_options):
```

Comment:
Fair enough. I don't want to nitpick this any further; this is still an improvement over the current code and I'd rather focus our efforts on other things. As always, NITs can be safely ignored and aren't blockers for merge.